src/collectd.conf.pod

```diff
@@ -10596,6 +10596,141 @@ C<https://monitoring.googleapis.com/v3>.
 
 =back
 
+=head2 Plugin C<write_syslog>
+
+The C<write_syslog> plugin writes data in I<syslog> format log messages.
+It implements the basic syslog protocol, extends it with
+content-based filtering, rich filtering capabilities,
+flexible configuration options and adds features such as using TCP for transport.
+The plugin can connect to a I<Syslog> daemon that will ingest metrics, transform and
+ship them to the specified output.
+The plugin uses I<TCP> over the "line based" protocol with a default port 44514.
```

Comment:
is there an RFC somewhere that describes this protocol, or another reference?

Reply:
Copied from tsdb ..

Reply:
we should definitely mention RFC5424

Reply:
fixed